pkg/network/tracer/connection/tracer.go

```diff
@@ -516,6 +529,10 @@ func (t *tracer) Collect(ch chan<- prometheus.Metric) {
 	delta = int64(ebpfTelemetry.Udp_dropped_conns) - ConnTracerTelemetry.lastUdpDroppedConns.Load()
 	ConnTracerTelemetry.lastUdpDroppedConns.Store(int64(ebpfTelemetry.Udp_dropped_conns))
 	ch <- prometheus.MustNewConstMetric(ConnTracerTelemetry.UdpDroppedConns, prometheus.CounterValue, float64(delta))
+
+	delta = int64(ebpfTelemetry.Tcp_dropped_conns) - ConnTracerTelemetry.lastTcpDroppedConns.Load()
```

Comment:
I think this has to be added to `Describe()` too... looking at it I think it's missing most of the metrics for this type

Reply:
fixed the other ones: https://github.com/DataDog/datadog-agent/pull/18314

Reply:
Added: https://github.com/DataDog/datadog-agent/pull/18268/commits/653b5792a3b819e85ab169624873e3e222f5e761